src/xscen/biasadjust.py

```diff
@@ -271,13 +288,26 @@ def adjust(
         dtrain.attrs["train_params"] = eval(dtrain.attrs["train_params"])  # noqa: S307
 
     var = dtrain.attrs["train_params"]["var"]
-    if len(var) != 1:
-        raise ValueError(
-            "biasadjust currently does not support entries with multiple variables."
-        )
-    else:
+    if len(var) == 1:
         var = var[0]
         sim = dsim[var]
+    else:
+        sim = xsdba.stack_variables(dsim[var])
+
+    # Used in MBCn adjusting
+    # I'm just assuming that if `ref` is needed, so is `hist`
+    if dref is not None:
+        train_period = dtrain.attrs["train_params"]["period"]
+        ref = xsdba.stack_variables(dref[var])
+        ref = ref.sel(time=slice(*train_period))
+        hist = sim.sel(time=slice(*train_period))
+        if (hist.time.size != ref.time.size) or (ref.time.size == 0):
+            raise ValueError(
```

Comment:
this needs to be checked after the calendar conversion. If ref is noleap and sim is standard, this error will be raised, but it shouldn't becuase the calendars will be converted to be the same.